integration/test/basic/basic_test.go

```diff
@@ -3,17 +3,82 @@
 package basic
 
 import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
 	"testing"
-)
+	"time"
 
-const (
-	chartOperatorValues = ``
+	"github.com/cenkalti/backoff"
+	"github.com/giantswarm/e2e-harness/pkg/framework"
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/microerror"
 )
 
-func TestInstallOperator(t *testing.T) {
-	err := f.InstallOperator("chart-operator", "chartconfig", chartOperatorValues, ":${CIRCLE_SHA1}")
+func TestChartInstalled(t *testing.T) {
+	err := installChartOperatorResource(f)
+	if err != nil {
+		t.Fatalf("could not install chart-operator-resource-chart %v", err)
+	}
+
+	var rc *helmclient.ReleaseContent
+	operation := func() error {
+		rc, err = helmClient.GetReleaseContent("tb-release")
+		if err != nil {
+			return fmt.Errorf("could not retrieve release content: %v", err)
```

Comment:
You can also use `microerror.Newf(...)` here, this way it is more consistent IMO.

Reply:
Sure, done.